index.html

```diff
@@ -1,16 +1,269 @@
+                    Material for MkDocs
+                </a>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-copyright">
+                Copyright © 2021 Eclipse Foundation<br>
+                <a href="https://www.eclipse.org/legal/privacy.php" target="_blank">Privacy policy</a><br>
+                <a href="https://www.eclipse.org/legal/termsofuse.php" target="_blank">Terms of use</a><br>
+                <a href="https://www.eclipse.org/legal/copyright.php" target="_blank">Copyright</a><br>
+                <a href="https://www.eclipse.org/legal/" target="_blank">Legal</a><br>
+                <a href="https://www.eclipse.org/security/" target="_blank">Report a safety or security
+                    vulnerability</a>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-copyright">
+                Sponsored by<br>
+                <a href="https://www.eclipse.org/" target="_blank" rel="noopener" title="Eclipse"><img
+                        src="./images/eclipse-logo.svg" width="100px" style="margin-top: 0.5rem"></img></a><br>
+                <a href="https://www.bosch.com/" target="_blank" rel="noopener" title="Robert Bosch GmbH"><img
+                        src="./images/bosch-logo.png" width="100px" style="margin-top: 0.5rem"></img></a><br>
+                <a href="https://www.apex.ai/" target="_blank" rel="noopener" title="Apex.AI Inc."><img
+                        src="./images/apex-logo.webp" width="100px" style="margin-top: 0.5rem"></img></a><br>
+                <a href="https://www.ekxide.io/" target="_blank" rel="noopener" title="ekxide IO GmbH"><img
+                        src="./images/ekxide-logo.svg" width="100px" style="margin-top: 0.5rem"></img></a><br>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-social">
+                <a href="https://github.com/eclipse-iceoryx/iceoryx" target="_blank" rel="noopener" title="GitHub iceoryx"
+                    class="md-social__link">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2A10 10 0 0 0 2 12c0 4.42 2.87 8.17 6.84 9.5.5.08.66-.23.66-.5v-1.69c-2.77.6-3.36-1.34-3.36-1.34-.46-1.16-1.11-1.47-1.11-1.47-.91-.62.07-.6.07-.6 1 .07 1.53 1.03 1.53 1.03.87 1.52 2.34 1.07 2.91.83.09-.65.35-1.09.63-1.34-2.22-.25-4.55-1.11-4.55-4.92 0-1.11.38-2 1.03-2.71-.1-.25-.45-1.29.1-2.64 0 0 .84-.27 2.75 1.02.79-.22 1.65-.33 2.5-.33.85 0 1.71.11 2.5.33 1.91-1.29 2.75-1.02 2.75-1.02.55 1.35.2 2.39.1 2.64.65.71 1.03 1.6 1.03 2.71 0 3.82-2.34 4.66-4.57 4.91.36.31.69.92.69 1.85V21c0 .27.16.59.67.5C19.14 20.16 22 16.42 22 12A10 10 0 0 0 12 2z"/></svg>
+                </a>
+                <a href="https://github.com/eclipse-iceoryx/iceoryx2" target="_blank" rel="noopener" title="GitHub iceoryx2"
```

Comment:
Using the same logo is confusing. Could we use a mini iceoryx logo here with a number?

Reply:
I'll play around with it.

I might end up having a vertical list with text along the icons

Reply:
Ok, if you change it, please at the same time also change it [here](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/overrides/partials/footer.html).